README.md

```diff
@@ -2,55 +2,39 @@
 
 # Project Name
 
-> Description the project.
-
+> In this project i've build an online website for an imaginary conference. 
 
 ## Built With
 
-- Major languages
-- Frameworks
-- Technologies used
+- HTML
+- CSS
+- JavaScript
 
 ## Live Demo (if available)
 
-[Live Demo Link](https://livedemo.com)
-
+[Live Demo Link](https://karayamanemre.github.io/coffee-summit/)
 
```

Comment:
## [Video Presentation is Missing]
1. Please record a video presenting the features of the project.
2. The maximum length of the video is 5 minutes.
3. The video should contains a clear and objective description about the project in professional English. The topic and purpose of the project are made clear.
4. The video contains a demo of the project, showing the pages in different screen sizes.
5. The video contains a highlight about a piece of the code the student finds interesting or is proud of making.